Pipelines/Pipeline.yml

```diff
@@ -0,0 +1,28 @@
+name: CD-ApiMDeveloperPortal-Build
```

Comment:
There is common practice to put all pipeline files into `.pipeline` folder. Also, I would give the file specific name, i.e. `migrate.yml`.

Reply:
updated.